symphony/lib/toolkit/class.entrymanager.php

```diff
@@ -174,39 +174,48 @@ public static function edit(Entry $entry)
                 continue;
             }
 
+            $did_lock = false;
             try {
+                $did_lock = Symphony::Database()->query('LOCK TABLES tbl_entries_data_' . $field_id . ' WRITE');
                 Symphony::Database()->delete('tbl_entries_data_' . $field_id, sprintf("
```

Comment:
From what I am reading: 

> No other session can access it until the lock is released.

This is then an exclusive lock. Other operation are "queued". And yes, this is what we need ;)